TTS/tts/models/vits.py

```diff
@@ -1108,8 +1154,14 @@ def train_step(self, batch: dict, criterion: nn.Module, optimizer_idx: int) -> T
 
             # compute melspec segment
             with autocast(enabled=False):
+
+                if self.args.TTS_part_sample_rate:
```

Comment:
You can also set this in the upsampling function I ask you to implement above. This way we keep all the related code in a single unit.

Reply:
Done